python/docs/source/getting_started/quickstart_df.ipynb

```diff
@@ -1167,7 +1167,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.8"
+   "version": "3.7.10"
```

Comment:
Out of curiosity, why do we upgrade the `version`?

Reply:
Seems like Binder upgraded their Python version .. I executed this notebook after my change to make sure it's working :-) ..

Reply:
Got it, thanks! :)